### PR TITLE
update

### PR DESCRIPTION
**Pull request by jmg3156**

### CODE REVIEW

File: js-intro.js [hunk 1]
```diff
@@ -2,61 +2,105 @@
...
+//var mantra = "B"
 
 // 1. Write the code that returns 'B' from mantra.
+   // console.log(mantra)
+
 
 // 2. Write the code that determines if there is a 'x' in mantra.
 
+//var mantra = "Be the Dev"
+//function mantra (){
```
Comment (JzGo) [inline js-intro.js]: Good job so far! You are very close with this solution and the following one. A couple of tips that will help you moving forward are:
 1) get into the habit of naming things a bit more descriptively. In this case you want to distinguish your method from the variable that it works with
2) get into the habit of passing variables as arguments as opposed to manipulating them with a direct call inside the function.

so, try renaming the mantra function and refactor it to take a variable like mantra as an argument

File: js-intro.js [hunk 2]
```diff
@@ -2,61 +2,105 @@
...
+  //  }
+
+ //   console.log(mantra.indexOf("v"))
+
 
 // Consider the following statement:
 
-var codingIsEasy
-var imBored
+//var codingIsEasy
```
Comment (JzGo) [inline js-intro.js]: For this and the next exercise, assign a boolean value to the variables for the right block to run

**Review by JzGo — COMMENTED** [review]
Your md looks great! I wanted to give you a chance though to have a second look at the js. I left some comments there to get you started. Take a stab at refactoring your other answers. When you've pushed your changes, go ahead and slack me.

**Review by sjproctor — APPROVED** [review]
Looks good Jessa, go ahead and merge your code and delete the branch!

**Pull request merged**